app/src/main/java/com/example/background/BlurApplication.kt

```diff
@@ -17,10 +17,24 @@
 package com.example.background
 
 import android.app.Application
+import androidx.work.Configuration
 import timber.log.Timber
 import timber.log.Timber.DebugTree
 
-class BlurApplication() : Application() {
+class BlurApplication() : Application(), Configuration.Provider {
+
+    override fun getWorkManagerConfiguration(): Configuration {
+        return if (BuildConfig.DEBUG) {
+            Configuration.Builder()
+                    .setMinimumLoggingLevel(android.util.Log.DEBUG)
```

Comment:
Import android.util.Log.

Reply:
Thanks,
Added a fix for the Logs import.